docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -2509,7 +2509,48 @@
   },
   {
    "cell_type": "markdown",
-   "id": "499025cb",
```

Comment:
Minor: could we restore this line? It may seem dumb but it will probably save us some trouble later on in case some conflicts come up

Reply:
When you execute notebook cell id changes so it becomes difficult to keep this the same